docs/index.html

```
@@ -0,0 +1,878 @@
+<title>EC-CUBE 4.4 とAI時代のEC ― 店舗オーナー向け勉強会</title>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)